docs/source/concept_models/vivarium_sanofi_multiple_myeloma/concept_model.rst

```diff
@@ -288,62 +288,198 @@ See :ref:`multiple myeloma cause model<2019_cancer_model_multiple_myeloma>`
-disease characteristics. For simulants at a given age, sex, and race/ethnicity, 
-the choice of therapy is based on their GFR and cytogenetic risk. 
-
-**eGFR** has a continuous risk exposure by age, sex, and race/ethnicity. 
-Simulants will be assigned to a CKD stage based on their eGFR value. We consider 
-5 stages of CKD:
-
- - Stage 1: eGFR > 90 ml/min/1.73m^2
- - Stage 2: eGFR 60 to 90 ml/min/1.73m^2
- - Stage 3: eGFR 30 to 60 ml/min/1.73m^2
- - Stage 4: eGFR 15 to 30 ml/min/1.73m^2
- - Stage 5: eGFR < 15 ml/min/1.73m^2
-
-**Cytogenetic risk** is a binary risk factor. Simulants will fall into one of 
-two risk exposure categories: with high-risk cytogenetics, or with standard-risk 
-cytogenetics. We intend to use Flatiron data to inform the existing prevalence 
-of high-risk cytogenetics among adults in the US population with multiple myeloma.
-
-.. list-table:: Cytogenetic risk stratification of myeloma ([Rajan-and-Rajkumar-et-al-2015]_)
-   :header-rows: 1
+In this model, we implement risk exposures for simulants upon entry to the MM state of the multiple myeloma cause model. Simulants in the susceptible state will not have risk exposure values (aside from sex and age) or risk effects. The risk exposure and effects for the risks included in this model (sex, age, race, cytogenetic risk, renal impairment) are described below. 
+
+.. _mm5.3.2.1:
+
+5.3.2.1 Risk Factor Exposure Initialization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Upon diagnosis with multiple myeloma, simulants should be assigned values for each of the following characteristics, with the probability shown in the table below. The probability of these exposures were obtained from Flatiron data reported by [Braunlin-et-al-2021]_. Notably, age and sex are included in this table for use in calculations described later in this document, but they do not need to be assigned to a simulant upon initialization, as each simulant already has a value for age and sex. For now, we will assume that each of the attributes are independent of one another. 
+
+Notably, we only have risk factor exposure distribution data among newly diagnosed patients. Due to the differential survival rates among the different risk exposure groups, we do not expect that the risk exposure distribution among relapsed and refractory multiple myeloma patients to be the same as among newly diagnosed patients. Due to our lack of data to inform risk exposure initialization probabilities among RRMM patients at the beginning of the simulation, we will model a "burn-in" period prior to the official time-frame of the simulation (2021-2026) in which all MM patients are initialized to the first MM state, allowing these risk exposure distributions to shift along with disease progression according to the risk effects described in the next section.
+
+.. todo::
+
+  Add more details on the burn-in period... to be expanded upon in the retreatment section
```

Comment:
Looking forward to more details on the burn-in period. Would we want this in the main sim or is this something we could maybe pre-compute?

Reply:
I suggest pre-compute and save the distribution of risk prevalence from different MM states. Then we can iteratively use the the risk exposure outcomes.

Reply:
I think that the initial idea was that we could run a burn in period, record information from it, and then not have to keep running it and initialize using that recorded information instead. I believe that it has already been performed for the cause model to calibrate the initial prevalence of each of the RRMM states (Yongquan/Abie/James worked on this), but I think we will have to do it again after the incorporation of this new information. Anyway, I will be working on this today so we should have more details soon!